.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,20 @@
----
-Make sure to mark the boxes below before creating PR: [x]
+<!--
+Thank you for contributing! Please make sure that your code changes
+are covered with tests. And in case of new features or big changes
+remember to adjust the documentation.
+
+Feel free to ping committers for the review!
 
-- [ ] Description above provides context of the change
-- [ ] Unit tests coverage for changes (not needed for documentation changes)
-- [ ] Target Github ISSUE in description if exists
-- [ ] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
-- [ ] Relevant documentation is updated including usage instructions.
-- [ ] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).
+In case of existing issue, reference it using one of the following:
+
+closes: #ISSUE
+related: #ISSUE
```

Comment:
Maybe someone change the comment directly, in this case can't target to issue, how could we avoid it?

Reply:
The description can always be updated so there should be no problem. Also we as committers should pay attention to link PR with issues so the auto-close feature works